templates/typescript/samples/sample-skill/src/index.ts

```diff
@@ -104,7 +105,7 @@ const supportedLocales: string[] = ['en-us', 'de-de', 'es-es', 'fr-fr', 'it-it',
 supportedLocales.forEach((locale: string): void => {
     const localeTemplateFiles: string[] = [];
     templateFiles.forEach((template: string): void => {
-        // LG template for default locale should not include locale in file extension.
+        // LG template for en-us does not include locale in file extension.
         if (locale === (botSettings.defaultLocale || 'en-us')) {
```

Comment:
```suggestion
        if (locale === 'en-us') {
```

Following [this](https://github.com/microsoft/botframework-solutions/blob/v0.8-beta/samples/csharp/skill/SkillSample/Startup.cs#L111)